cpp/perspective/CMakeLists.txt

```diff
@@ -679,6 +688,7 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
             target_compile_options(psp PRIVATE -fvisibility=hidden)
             # target_compile_options(psppy PRIVATE -fvisibility=hidden)
         else()
+            target_compile_options(psp PRIVATE -fvisibility=hidden)
```

Comment:
why is this flag set for `MANYLINUX`? can you identify what cases this `else()` targets?

Reply:
This was for emscripten, I can add another `elseif` for manylinux explicitly. Good catch 👍